### DISCUSSION TITLE
New Device Tracker

### DISCUSSION THREAD

**Discussion by g-off-white** [travisghansen/hass-pfsense#137] [Q&A] [answered]
How do you add/remove devices from the pfsense tracker after initial configuration?

**Comment by travisghansen** [comment, marked as answer]
Just hit configure on the integration and you can add/remove devices.

**Comment by g-off-white** [comment]
Thanks. I saw the first submit button and assumed that was it.

Regards,

Geoff white
-------------------
Disclaimer: I am blaming any typos in this email on my phone.
Please do not mistake my brevity as rudeness; my rudeness is unmistakable.

On 28 Jan 2023, at 12:34 am, Travis Glenn Hansen ***@***.***> wrote:

﻿

Just hit configure on the integration and you can add/remove devices.

**Comment by LambMediaGroup** [comment]
For some reason I had no option to add a device to the list of tracked devices. I had to disable device tracking click submit and then enable device tracking again and submit. Then it gave me a list to choose from. Seems a poor implementation to me. Can devices to track not just be an option on the main configure screen as I spent ages trying to find this option that's basically hidden after the initial set up.
